tests/foreman/ui/test_lifecycleenvironment.py

```diff
@@ -33,6 +39,45 @@ def setUpClass(cls):
         super(LifeCycleEnvironmentTestCase, cls).setUpClass()
         cls.org_name = entities.Organization().create().name
 
+    def _create_role_permissions(self, role, permissions_types_names):
+        """Create role permissions found in dict permissions_types_names
+        :param role: nailgun.entities.Role
+        :param permissions_types_names: a dict containing permissions types
+               and permission names for example
+
+              ::
+
+               permissions_types_names = {
+                   None: ['access_dashboard'],
+                   'Organization': ['view_organizations'],
+                   'Location': ['view_locations'],
+                   'Katello::KTEnvironment': [
+                       'view_lifecycle_environments',
+                       'edit_lifecycle_environments',
+                       'promote_or_remove_content_views_to_environments'
+                   ]
+               }
+        """
+        for resource_type, permissions_name in permissions_types_names.items():
+            if resource_type is None:
+                permissions_entities = []
+                for name in permissions_name:
+                    permissions_entities.append(
+                        entities.Permission(name=name).search()[0])
```

Comment:
@rochacbruno moving this function to robottelo/api/tools, as it's often faced in tests, so we should have a standard way to add permissions to roles and do all the necessary checks there, what do you think?